### PR TITLE
CB-1: Cooking time

### PR DESCRIPTION
**Pull request by Narzew**

### CODE REVIEW

File: app/serializers/recipe_serializer.rb [hunk 1]
```diff
@@ -1,3 +1,12 @@
 class RecipeSerializer < ActiveModel::Serializer
-  attributes :name, :content
+  attributes :id, :name, :content, :cooking_time
+
+  def cooking_time
+    if object.cooking_time.blank?
+      "unknown"
+    else
+      object.cooking_time
+    end
```
Comment (akademia-knowde) [inline app/serializers/recipe_serializer.rb:9]:
```suggestion
      return  "unknown" unless object.cooking_time
  
      object.cooking_time
```

File: config/database.yml [hunk 2]
```diff
@@ -15,6 +15,7 @@
 # gem 'pg'
 #
 default: &default
+  username: niki
```
Comment (akademia-knowde) [inline config/database.yml:18]: unnecessary

File: .gitignore [hunk 3]
```diff
@@ -25,3 +25,4 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+/config/database.yml
```
Comment (akademia-knowde) [inline .gitignore:28]: missing new line char

**Review by akademia-knowde — CHANGES_REQUESTED** [review]
missing specs for serializer

File: app/serializers/recipe_serializer.rb [hunk 4]
```diff
@@ -1,3 +1,9 @@
 class RecipeSerializer < ActiveModel::Serializer
-  attributes :id, :name, :content
+  attributes :id, :name, :content, :cooking_time
+
+  def cooking_time
+    return "unknown" unless object.cooking_time
+    object.cooking_time
+  end
+
```
Comment (akademia-knowde) [inline app/serializers/recipe_serializer.rb:8]: unnecessary

File: app/serializers/recipe_serializer.rb [hunk 5]
```diff
@@ -1,3 +1,9 @@
 class RecipeSerializer < ActiveModel::Serializer
-  attributes :id, :name, :content
+  attributes :id, :name, :content, :cooking_time
+
+  def cooking_time
+    return "unknown" unless object.cooking_time
```
Comment (akademia-knowde) [inline app/serializers/recipe_serializer.rb:5]: guard clause - pusta linia

File: spec/serializers/recipe_serializer_spec.rb [hunk 6]
```diff
@@ -7,4 +7,16 @@
...
+      let(:recipe) { create(:recipe, cooking_time: nil) }
+      it { is_expected.to include(cooking_time: 'unknown') }
+    end
+    context 'with cooking_time present' do
+      let(:recipe) { recipe = FactoryBot.build(:recipe) }
+      it { is_expected.to include(cooking_time: recipe.cooking_time) }
+    end
+  end
+
+end
```
Comment (akademia-knowde) [inline spec/serializers/recipe_serializer_spec.rb:22]: newline

File: spec/serializers/recipe_serializer_spec.rb [hunk 7]
```diff
@@ -7,4 +7,16 @@
...
+    context 'with cooking_time blank' do
+      let(:recipe) { create(:recipe, cooking_time: nil) }
+      it { is_expected.to include(cooking_time: 'unknown') }
+    end
+    context 'with cooking_time present' do
+      let(:recipe) { recipe = FactoryBot.build(:recipe) }
+      it { is_expected.to include(cooking_time: recipe.cooking_time) }
+    end
+  end
+
```
Comment (akademia-knowde) [inline spec/serializers/recipe_serializer_spec.rb:21]: unnecessary

File: spec/serializers/recipe_serializer_spec.rb [hunk 8]
```diff
@@ -7,4 +7,16 @@
 
   it { is_expected.to include(name: recipe.name) }
   it { is_expected.to include(content: recipe.content) }
-end
+
+  describe 'cooking_time' do
+    context 'with cooking_time blank' do
+      let(:recipe) { create(:recipe, cooking_time: nil) }
+      it { is_expected.to include(cooking_time: 'unknown') }
+    end
```
Comment (akademia-knowde) [inline spec/serializers/recipe_serializer_spec.rb:15]: empty line

**Review by akademia-knowde — CHANGES_REQUESTED** [review]

**Review by h4b00 — COMMENTED** [review]
please Add back `config/database.yml ` it should be in gitignore but it shouldn't be deleted from the repo.
rest work looks good 💯 !

**Comment by Narzew** [comment]
> please Add back `config/database.yml ` it should be in gitignore but it shouldn't be deleted from the repo.
> rest work looks good  !

Done, I'm waiting for merge

**Review by akademia-knowde — APPROVED** [review]

**Pull request merged**